include/SDL3/SDL_audio.h

```diff
@@ -1031,33 +1031,55 @@ extern DECLSPEC void SDLCALL SDL_DestroyAudioStream(SDL_AudioStream *stream);
 
 
 /**
- * Convenience function to create and bind an audio stream in one step.
+ * Convenience function for straightforward audio init for the common case.
  *
- * This manages the creation of an audio stream, and setting its format
- * correctly to match both the app and the audio device's needs. This is
- * optional, but slightly less cumbersome to set up for a common use case.
+ * If all your app intends to do is provide a single source of PCM audio,
+ * this function allows you to do all your audio setup in a single call.
+ *
+ * This is intended to be a clean means to migrate apps from SDL2.
+ *
+ * This function will open an audio device, create a stream and bind it.
+ * Unlike other methods of setup, the audio device will be closed when this
+ * stream is destroyed, so the app can treat the returned SDL_AudioStream
+ * as the only object needed to manage audio playback.
+ *
+ * Also unlike other functions, the audio device begins paused. This is
+ * to map more closely to SDL2-style behavior, and since there is no extra
+ * step here to bind a stream to begin audio flowing.
+ *
+ * This function works with both playback and capture devices.
  *
  * The `spec` parameter represents the app's side of the audio stream. That
  * is, for recording audio, this will be the output format, and for playing
- * audio, this will be the input format. This function will set the other side
- * of the audio stream to the device's format.
- *
- * \param devid an audio device to bind a stream to. This must be an opened
- *              device, and can not be zero.
- * \param spec the audio stream's input format
- * \returns a bound audio stream on success, ready to use. NULL on error; call
- *          SDL_GetError() for more information.
+ * audio, this will be the input format.
+ *
+ * If you don't care about opening a specific audio device, you can (and
+ * probably _should_), use SDL_AUDIO_DEVICE_DEFAULT_OUTPUT for playback and
+ * SDL_AUDIO_DEVICE_DEFAULT_CAPTURE for recording.
+ *
+ * One can optionally provide a callback function; if NULL, the app is
+ * expected to queue audio data for playback (or unqueue audio data if
+ * capturing). Otherwise, the callback will begin to fire once the device is
+ * unpaused.
+ *
+ * \param devid an audio device to open, or SDL_AUDIO_DEVICE_DEFAULT_OUTPUT
+ *              or SDL_AUDIO_DEVICE_DEFAULT_CAPTURE.
+ * \param spec the audio stream's input format. Required.
```

Comment:
the audio stream's data format (?)

Reply:
Fixed in 01c93425849fe7f0b1bc96610a312ec8a58d943b